### ISSUE TITLE
Altis CMS Build Failing

### ISSUE THREAD

**Issue by robindevitt** [humanmade/altis-cms#508]
On PR [#506](https://github.com/humanmade/altis-cms/pull/506) it was identified during the checks the [Travis CI build](https://github.com/humanmade/altis-cms/pull/506/checks?check_run_id=5006254989) failed. 

Initial investigation, it appears to be related to Authorship.

A previus build from the same issue/pr had this: [build report](https://app.travis-ci.com/github/humanmade/altis-cms/builds/245546334)

**Comment by robindevitt** [comment]
Failing with: AuthorshipCest: Test I can add multiple authors

Scenario Steps:
`$I->seePostWithTermInDatabase("5",false,null,"auth...")` [Line 115](https://github.com/humanmade/altis-cms/blob/26d4006aafe91a82311a3b1db63e344085fbdb27/tests/acceptance/AuthorshipCest.php#L115)

https://github.com/humanmade/altis-cms/blob/26d4006aafe91a82311a3b1db63e344085fbdb27/tests/acceptance/AuthorshipCest.php#L107-L115

`$author_term_id` is returning no value and been seen as false based on this : 
`$I->seePostWithTermInDatabase("5",false,null,"auth...")`

**Comment by robindevitt** [comment]
Hey @shadyvb do you have any suggestions/direction as to what may be happening here and causing the fail?

**Comment by robindevitt** [comment]
Slack thread : https://hmn.slack.com/archives/C5WSNGC9F/p1644223347090779
Tl;dr -> The symptom here is that the authorship term is not being created when the user is selected from the dropdown.

**Comment by roborourke** [comment]
The other linked build failure is likely related to this output further up the logs, not entirely sure why it failed at that stage right now:

```
ERROR: for kibana  Container "f7da2e5bb2a6" is unhealthy.
Encountered errors while bringing up the project.
Services failed to start successfully.
The command "cd $HOME/test-root && composer serve" exited with 1.
134.83s$ cd $HOME/test-root && composer dev-tools codecept run -p vendor/$ALTIS_PACKAGE/tests
Running 1 suite(s), (acceptance)..
Running "acceptance" test suite..
Error: The site you have requested is not installed.
Run `wp core install` to create database tables.
```

**Comment by shadyvb** [comment]
Moved the reindex function into the shared helper class, and issued pull-requests for the changes.
